internal/apis/certmanager/validation/issuer_test.go

```diff
@@ -737,6 +737,30 @@ func TestValidateIssuerSpec(t *testing.T) {
 				field.Invalid(fldPath.Child("ca", "ocspServer").Index(0), "", `must be a valid URL, e.g., http://ocsp.int-x3.letsencrypt.org`),
 			},
 		},
+		"valid IssuingCertificateURLs": {
+			spec: &cmapi.IssuerSpec{
+				IssuerConfig: cmapi.IssuerConfig{
+					CA: &cmapi.CAIssuer{
+						SecretName:             "valid",
+						IssuingCertificateURLs: []string{"http://ca.domain.org/ca.crt"},
```

Comment:
nitpick (non-blocking): I tend to prefer using one of the [reserved example domains](https://www.rfc-editor.org/rfc/rfc2606.html#section-3) for tests, but it doesn't really make any difference here!

```suggestion
						IssuingCertificateURLs: []string{"http://ca.example.com/ca.crt"},
```

Reply:
Changed